### PR TITLE
Added missing options for infinite scroll

### PR DESCRIPTION
**Pull request by ashwin027**
infiniteScrollRowsFromEnd, infiniteScrollUp and infiniteScrollDown were
missing in IGridOptions. These were added.

### CODE REVIEW

**Comment by vvakame** [comment]
*ui-grid/ui-grid.d.ts*

to authors(@btesser @joeskeen). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

File: ui-grid/ui-grid.d.ts [hunk 1]
```diff
@@ -2087,6 +2087,25 @@ declare module uiGrid {
              * @default true
              */
             enableInfiniteScroll?: boolean;
+            /**
```
Comment (joeskeen) [inline ui-grid/ui-grid.d.ts]: It seems a little strange to me, but in the documentation for the infiniteScroll module it doesn't have these options listed in the GridOptions for the module (see http://ui-grid.info/docs/#/api/ui.grid.infiniteScroll.api:GridOptions).  Instead, I see these three options listed in the base/global module for ui-grid (see http://ui-grid.info/docs/#/api/ui.grid.class:GridOptions).  So perhaps these options should be moved to `uiGrid.IGridOptions`.  Aside from that, I'm OK with this change.

Comment (ashwin027) [inline ui-grid/ui-grid.d.ts]: @joeskeen You are right...I've moved it out into the global grid options. I've committed my changes to my branch but they aren't showing up in the PR yet. I'm new to Github so whats the process now? Should I submit a separate PR?

Thanks for the review and thanks in advance for the help! :)

Comment (joeskeen) [inline ui-grid/ui-grid.d.ts]: @ashwin027 it looks like your pull request has been updated automatically.

**Comment by joeskeen** [comment]
@ashwin027 this looks good to me! :+1: 

@vvakame ready to merge!

File: ui-grid/ui-grid.d.ts (hunk 1 above)
Comment (ashwin027) [inline ui-grid/ui-grid.d.ts]: @joeskeen It took an awfully long time to show up. Thanks for the review!

Comment (joeskeen) [inline ui-grid/ui-grid.d.ts]: Thanks for your contribution!

**Comment by vvakame** [comment]
thanks all!
